### PR TITLE
Create page about updates

### PR DESCRIPTION
**Pull request by MichaelDeciso**

### CODE REVIEW

File: source/manual/updates.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+=======
+Updates
+=======
+
+OPNsense's update schedule consists of two major releases each year, which are updated every two weeks. The major
```
Comment (mimugmail) [inline source/manual/updates.rst]:
```suggestion
OPNsense's update schedule consists of two major releases each year, which are updated about every two weeks. The major
```

Comment (mimugmail) [reply source/manual/updates.rst]: This sound very fixed which is not.

**Comment by MichaelDeciso** [comment]
@AdSchellevis I solved the merge conflict, could you merge this?

**Comment by AdSchellevis** [comment]
@MichaelDeciso please fix @mimugmail objection first, the "about"  doesn't appear to be in the text now.

**Comment by MichaelDeciso** [comment]
You're right, it got lost in the rebase. Re-added it.

**Pull request merged**